viewer/src/views/threejs/cad/matCapTextureData.ts

```diff
@@ -0,0 +1,5 @@

```

Comment:
Add a reference to https://github.com/nidorx/matcaps as source of the image (https://github.com/nidorx/matcaps/blob/master/PAGE-26.md)

Reply:
Also, size is 232x232 and ThreeJS reduces size to 128x128 runtime. Reduce size of the encoded image to 128x128.